bench_test.go

```diff
@@ -138,6 +138,31 @@ func BenchmarkLogLineParse(b *testing.B) {
 	sink = ll
 }
 
+func BenchmarkLogLineToText(b *testing.B) {
+	b.Run("from set", func(b *testing.B) {
+		ll := baker.LogLine{FieldSeparator: 44}
+		for i := 0; i < 100; i++ {
+			ll.Set(baker.FieldIndex(i), []byte("value"))
+		}
+
```

Comment:
Maybe you should reset the timer. You are not benchmarking the `ll.Set` operation, but the `ll.ToText`.
Same for `"from parse"`.

Reply:
These two benchmarks are made to access the performance of only the `ToText` method. But the `ToText` has a fast path in the situation in which the `Set` method is never called and a slow path when at least one time the `Set` method is called.  Thus, this is the reason for having these two benchmarks.